mod-record-specifications-server/src/test/java/org/folio/api/SpecificationStorageIndicatorsApiIT.java

```diff
@@ -61,6 +68,20 @@ static void beforeAll() {
     setUpTenant();
   }
 
+  @BeforeEach
+  void setUp(@Autowired KafkaProperties kafkaProperties) {
+    consumerRecords = new LinkedBlockingQueue<>();
+    container =
+      createAndStartTestConsumer(specificationUpdatedTopic(),
+        consumerRecords, kafkaProperties, SpecificationUpdatedEvent.class);
+  }
+
+  @AfterEach
+  void tearDown() {
+    consumerRecords.clear();
+    container.stop();
+  }
+
   @Test
```

Comment:
Why don't have it IntegrationBase?

Reply:
I assume we will have validation API in a future which will not require kafka container in tests

Reply:
Okay, but IntegrationBase was updated with some Kafka logic. Maybe then it make sense to create additional child for IntegrationBase that will work with kafka, and then it could be used as parent for ITs that work with Kafka